package/pom.xml

```diff
@@ -61,22 +43,21 @@
     <appframeworkVersion>2.16.0</appframeworkVersion>
 
     <!-- OpenMRS Core -->
-    <openMRSVersion>2.3.3</openMRSVersion>
-
+    <openMRSVersion>2.4.0</openMRSVersion>
   </properties>
 
   <dependencies>
     <dependency>
-      <groupId>org.openmrs</groupId>
+      <groupId>${project.groupId}</groupId>
       <artifactId>openmrs-config-refapp</artifactId>
       <type>zip</type>
-      <version>${openmrsConfigVersion}</version>
+      <version>${project.version}</version>
```

Comment:
The reason I didn't use the same versions for Maven modules is that I thought we may ultimately want to remove the configs from this project and have them to live on their own repo.

That way if one just wants to change a single piece of metadata, only the OpenMRS config will have to released. SPA, SPA configs and other artifacts won't need to be re-released/re-deployed.

But I guess it doesn't make sense if they are all in the same repo.

Reply:
Yeah... to be honest, I was a little unsure about this, because there could be real value in versioning the metadata separately from the plumbing and certainly for something other than the RefApp.